components/Video/Video.js

```diff
@@ -50,6 +50,10 @@ class Video extends React.Component {
       uri: PropTypes.string,
     }),
     style: PropTypes.object,
```

Comment:
Same comment as above.

Reply:
Which type of `style` is right as your opinion, object or number?
If I create a style like use `StyleSheet.create`, It will return a number.

Reply:
Your prop types are OK, but you didn't remove the old style declaration, so you now have this:
```
style: PropTypes.object, // remove this
style: PropTypes.oneOfType([
  PropTypes.object,
  PropTypes.number,
]),
```
Just remove the first one, please.